kube/services/fence/fence-canary-service.yaml

```diff
@@ -0,0 +1,17 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: fence-canary-service
+spec:
+  selector:
+    app: fence-canary
```

Comment:
This should be
```
app: fence
release: canary
```
- probably need to add a `release` to the default service too ...